packages/react-ui/src/hooks/useFilter.ts

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react"
+import type { Filter } from "@hatchifyjs/rest-client/dist/services"
```

Comment:
this import isn't correct, you'll need to export it in `packages/rest-client/rest-client.ts` then you can properly import it